packages/titus-backend-typescript/lib/plugins/auth/azure-ad/azure-ad.test.js

```diff
@@ -0,0 +1,325 @@
+import fastify from 'fastify'
```

Comment:
I wanted to port this file to TS as well, but decided against it in the end. Typing mocks is always very hard and for specs it doesn't really mattter-you still benefit from TS types even in JS files for IDE autocomplete and such.

Reply:
yes that was the first thing I have tried. Still even when I used that, there are like 12 places where typescript is screaming with errors. I will publish it into a branch for you to take a look